server/gokbg3/grails-app/controllers/org/gokb/rest/CuratoryGroupsController.groovy

```diff
@@ -287,7 +287,6 @@ class CuratoryGroupsController {
       }
       else{
         try{
-          superordinate.subordinatedGroups << subordinate
```

Comment:
I think we need this for "re-escalation" (a superordinated CG giving the issue back to the subordinated CG) . Don't we?

Reply:
I moved this out of here since this is an unrelated issue.